components/event-bus/internal/trace/tags.go

```diff
@@ -11,6 +11,15 @@ const (
 	SubscriptionEnvironment = "sub-env"
 )
 
+const (
+	// push request headers to endpoint
+	HeaderSourceID         = "Ce-Source-ID"
```

Comment:
Does it make sense to make all header values lower-case? (e.g **ce-source-id**).

Reply:
We talked this with @radufa. This needs to be discussed and changed all across our components. We can create a follow-up issue for this.

Reply:
The headers are case insensitive and we have used "camel case" in Event-Bus in the past.